Gemfile

```diff
@@ -38,7 +38,7 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   # Use sqlite3 as the database for Active Record
-  gem 'sqlite3' , '~> 1.3.13'
+  gem 'sqlite3' , git: "https://github.com/sparklemotion/sqlite3-ruby"
```

Comment:
Why? Also, is this the default Gemfile?

Reply:
No we used that source because i had an error showing only in windows . That solution works for mac also so i left it like that . No the default is  '~> 1.4.0' or `~>1.13.3` depending on what version is installed on the machine .